app/js/directives/yesNoQuestion.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+angular.module('ddsApp').directive('yesNoQuestion', function() {
+    return {
+        restrict: 'E',
+        transclude: true,
+        templateUrl: 'partials/foyer/yes-no-question.html',
+        scope: true,
+        controller: 'yesNoQuestionCtrl',
+        link: function ($scope, $element, $attributes) {
+            $scope.modelString = $attributes.model;
+        }
+    };
+});
+
+angular.module('ddsApp').controller('yesNoQuestionCtrl', function($scope, $transclude, $element, $interpolate) {
+	_.forEach($transclude(), function(elem) {
+		if (elem.localName == 'question') {
+			$element.find('legend').append($interpolate(elem.innerHTML)($scope));
+		} else if (elem.localName == 'help-block') {
+			$element.find('p').append($interpolate(elem.innerHTML)($scope));
+		}
+	});
+
```

Comment:
The following is a little complex, but this is the best I found to be able to make this component able to take a `model` parameter. Any better implementation would be welcome, but I didn't find anything online.